src/System.Private.ServiceModel/src/System/IdentityModel/Selectors/KerberosSecurityTokenProvider.cs

```diff
@@ -59,16 +59,16 @@ public NetworkCredential NetworkCredential
             get { return _networkCredential; }
         }
 
-        internal SecurityToken GetToken(CancellationToken cancellationToken, ChannelBinding channelbinding)
+        internal SecurityToken GetToken(TimeSpan timeout, ChannelBinding channelbinding)
```

Comment:
Aren't these breaking changes?

Reply:
These are internal only methods. Although the class says it's public, it's not in our contract. We've got a lot of public implementations not in our contract because of copying code from .NET Framework.

---
In reply to: [274009300](https://github.com/dotnet/wcf/pull/3538#discussion_r274009300) [](ancestors = 274009300)